docs/guides/first-shopify-abtest.mdx

```diff
@@ -0,0 +1,140 @@
+
+Once your store is up and running, you'll want to log into your Shopify store as admin. You might already have done this and here's just a quick recap so we're all on the same page: 
+- Log into https://partners.shopify.com/ 
+- Click on **Stores** in the left navigation menu
+- Click on **Log in** link next to the store where you want to run the A/B test
+
+Here's my store on the Shopify Partners portal. It's called **Fall Colors**, where I sell photographs that I've taken in the fall season when everything's so colorful and beautiful. My store happens to still be in development stage.
+
+![image](https://user-images.githubusercontent.com/1315028/150030303-eb4068ee-f5b8-42bc-9f94-9ca17df36e17.png)
+
+- On the **admin** landing page, click on **Customize Theme** tab and click the **Customize theme** button shown below
+
+![image](https://user-images.githubusercontent.com/1315028/150182063-fdc2bc10-48e1-414f-a05a-dd32c55a376f.png)
+
+- Click on the **Customize** button 
+
+![image](https://user-images.githubusercontent.com/1315028/150182338-065de17e-a33d-4141-9b34-9518a53b60eb.png)
+
+- From the left hand navigation menu, click on **Image banner**
+
+![image](https://user-images.githubusercontent.com/1315028/150182526-dd77fea2-20b9-4812-b6f9-30ab309a6a49.png)
+
+- Under the **First Image**, click **Select Image** button 
+
+![image](https://user-images.githubusercontent.com/1315028/150182728-2cae4794-0a08-4781-9d5f-6cbfe7394d48.png)
+
+- Select the hero banner image that you want to use as control from the library or upload a new image
+
+![image](https://user-images.githubusercontent.com/1315028/150182819-5a6e1745-7e54-4d02-a3e4-406eb278da8d.png)
+
+- Repeat the previous two steps for the **Second Image** as well
+- At this point, you should see both images in your hero banner on the store 
+
+
+### Step 2: Create an experiment on Statsig
+
+To create an experiment on Statsig, 
+- Log into https://console.statsig.com/
+- Click on **Experiments** from the left hand navigation menu
+- Click on **Create**, then enter the name and description of the experiment
+- In the experiment **Setup** tab, enter the names of the variants that you want to test in the **Groups** panel; in this case I'm just calling them Control and Test. 
+- Staying in the **Groups** panel, click on **sample_parameter** and enter a parameter name of your choice; in this case, I'm calling the parameter **image_selector**. Enter the parameter values that you want to use; in this case, I simply want to toggle between the two images that we've already uploaded to the store in the previous step
+
+![image](https://user-images.githubusercontent.com/1315028/150181235-068a1cf3-00ef-48f8-b2fa-51c0338e5e6d.png)
+
+
+### Step 3: Customize your theme
```

Comment:
This is a super hacky way to do it, and run a specific test related to swapping images.  Worth noting that this doesnt just magically run any arbitrary experiments